src/dataexplorer/dataexplorer-application/src/components/PatientInfo/DisplayPatientInfo.tsx

```diff
@@ -33,51 +55,101 @@ const DisplayPatientInfo = (props: {
             <Row gutter={[1, 40]} justify="center">
                 <Col span={12}>
                     <Card
+                        key={props.patient.id}
                         className="patient-card"
+                        title={<Title level={4}>{name}</Title>}
+                        headStyle={{
+                            backgroundColor: '#f7fbff',
+                            display: 'flex',
+                            justifyContent: 'center',
+                            marginBottom: 0,
```

Comment:
Shouldn't this be in a seperate scss file?

Reply:
Yes. Fixed